<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewGUI</class>
 <widget class="QMainWindow" name="ViewGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2P Listen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
    <item>
     <widget class="QLabel" name="_labelYourAddress">
      <property name="text">
       <string>Your LAN Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Loading Address...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="listenerTab">
       <attribute name="title">
        <string>listen</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="portLayout">
          <item>
           <widget class="QLabel" name="_labelSelectPort">
            <property name="text">
             <string>Select Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="listenPortLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777205</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMask">
             <string>00000</string>
            </property>
            <property name="text">
             <string>45954</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="transfererTab">
       <attribute name="title">
        <string>transfer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="_labelSelectEndpoint">
            <property name="text">
             <string>Send to:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="endpointLayout" stretch="8,1,4">
            <item>
             <widget class="QLineEdit" name="addressLineEdit">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>0</height>
               </size>
              </property>
              <property name="inputMask">
               <string>000.000.000.000</string>
              </property>
              <property name="text">
               <string>192.168.1.57</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="_colon_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="transferPortLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777205</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="inputMask">
               <string>00000</string>
              </property>
              <property name="text">
               <string>45954</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="selectFileButton">
          <property name="text">
           <string>Select File...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="actionButton">
      <property name="text">
       <string>listen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>listenPortLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionButton</sender>
   <signal>clicked()</signal>
   <receiver>ViewGUI</receiver>
   <slot>action_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>ViewGUI</receiver>
   <slot>action_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectFileButton</sender>
   <signal>clicked()</signal>
   <receiver>ViewGUI</receiver>
   <slot>select_file_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_button_clicked()</slot>
  <slot>action_changed(int)</slot>
  <slot>select_file_button_clicked()</slot>
 </slots>
</ui>
